lib/puppet/vendor/semantic_puppet/lib/semantic_puppet.rb

```diff
@@ -1,14 +1,8 @@
-begin
-  require 'gettext-setup'
-rescue LoadError
-  def _(msg)
-    msg
-  end
-end
+require 'gettext-setup'
```

Comment:
It sure will. I changed the last commit to not require gettext-setup and instead rely on the constant {{Puppet::GETTEXT_AVAILABLE}}. When doing this, I ran some tests. Turns out that puppet cannot run without the gem anyway so I created ticket https://tickets.puppetlabs.com/browse/PUP-7480.